subsys/fb/cfb_shell.c

```diff
@@ -19,8 +19,11 @@
 #define HELP_INIT "call \"cfb init\" first"
 #define HELP_PRINT "<col: pos> <row: pos> \"<text>\""
 
+#ifdef CONFIG_CHARACTER_FRAMEBUFFER_SHELL_DRIVER_NAME
```

Comment:
Either remove `CONFIG_CHARACTER_FRAMEBUFFER_SHELL_DRIVER_NAME` completely or revert the changes and update the board specific configs i the samples.

Reply:
@vanwinkeljan , remove the CONFIG_CHARACTER_FRAMEBUFFER_SHELL_DRIVER_NAME seems the only option, as it is a choice option, which can not be put into board specific configs in the samples.

Reply:
As far as I can see `CHARACTER_FRAMEBUFFER_SHELL_DRIVER_NAME` is just a string config and not part of a choice. Could it be that you update all CFB samples and not only the shell one?

Reply:
@vanwinkeljan , only shell example using this way, all other cfb examples use the DISPLAY_DRIVER, I do not know why it is originally designed so. So you think it is OK for me to change it? let me have a try.

Reply:
Yes it forces the user to specify the display name to use for the shell. So can this change be reverted?

Reply:
yes